docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1137 @@
+{
```

Comment:
<p>Looks great! Easy to follow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>